dbms/src/Interpreters/QueryLog.cpp

```diff
@@ -20,11 +21,23 @@
 namespace DB
 {
 
+template <> struct NearestFieldTypeImpl<QueryLogElement::Type> { using Type = UInt64; };
+
 Block QueryLogElement::createBlock()
 {
+
+    auto query_status_datatype = std::make_shared<DataTypeEnum8>(
+        DataTypeEnum8::Values
+        {
+            {"QUERY_START",                   static_cast<Int8>(QUERY_START)},
```

Comment:
Look at trace_log to check for style consistency.

Reply:
I looked at part_log. https://github.com/yandex/ClickHouse/blob/6625536236947629b8919006515fcde6a4bfd4e7/dbms/src/Interpreters/PartLog.cpp#L25

Maybe It is better to rewrite both of them?

Reply:
Let's do it.

PS. `CamelCase` please, not `That case`.